fastlane_bot/events/utils.py

```diff
@@ -1284,14 +1271,12 @@ def get_latest_events(
 
     # Get all event filters, events, and flatten them
     events = [
-        complex_handler(event)
-        for event in [
-            complex_handler(event)
-            for event in get_all_events(
-                n_jobs,
-                get_event_filters(n_jobs, mgr, start_block, current_block),
-            )
-        ]
+        Event.from_dict(complex_handler(event))
+        for events in get_all_events(
```

Comment:
minor point -- but having the loop variable and the final output variable both called `events` is slightly confusing